src/Symfony/Bundle/Test/ApiTestCase.php

```diff
@@ -94,9 +94,17 @@ protected function findIriBy(string $resourceClass, array $criteria): ?string
             return null;
         }
 
+        return $this->getIriForResource($item);
+    }
+
+    /**
+     * Generate the IRI of a resource item.
+     */
+    protected function getIriForResource(object $resource): ?string
```

Comment:
what about using the same method name as for the iri converter?

Reply:
Fine by me!

Reply:
the test needs fixing then :)

Reply:
My bad 😅 fixed